en/docs/develop-components/configure-endpoints-body.md

```diff
@@ -0,0 +1,124 @@
+
+Choreo defines endpoints by combining port binding, protocol, endpoint name, network visibility, endpoint schema, and additional protocol-related fields. Following table describe each attribute of an endpoint.
+
+| Field | Description |
+| ----- | ----------- |
+| Name | A unique identifier for the endpoint within the service component. |
+| Port | The network port on which the endpoint is accessible. |
+| Type | The communication protocol used by the endpoint (e.g., HTTP, HTTPS, gRPC, etc.). |
+| Network Visibility | Determines the level of visibility of an endpoint. Possible values are: <ul><li>Project: Allows components within the same project to access the endpoint.</li><li>Organization: Allows any component within the same organization to access the endpoint but restricts access to components outside the organization.</li><li>Public: Allows any client to access the endpoint, regardless of location or organization.</li></ul> |
+| Schema | specifies the structure and format of the data exchanged through the endpoint. |
+| Context (HTTP and GraphQL only) | A context path that you add to the endpoint's URL for routing purposes. |
+
+## Configure endpoints
+
+The method of defining endpoints depends on the build preset. There are three build presets the supports defining endpoints: 
+
+- Ballerina
+- Dockerfile
+- WSO2 MI
+
+### Configure endpoints with the Dockerfile build-preset
+
+When you build a service component using the Dockerfile build-preset, you can configure the endpoint details with the endpoints.yaml configuration file. You must place this file inside the `.choreo` directory at the build context path and commit it to the source repository.
+
+See [Understanding the endpoints.yaml file](#understanding-the-endpointsyaml-file) to learn about the endpoints.yaml file.
+
+### Configure endpoints with the Ballerina build-preset
+
+When you create a service component with the `Ballerina preset`, Choreo automatically detects the endpoint details for REST API and GraphQL endpoints. You can override the auto-generated endpoint configuration by providing the endpoints.yaml file in the source directory
+
+See [Understanding the endpoints.yaml file](#understanding-the-endpointsyaml-file) to learn about the endpoints.yaml file.
+
+
+### Configure endpoints with the WSO2 MI build-preset
+
+WSO2 MI build preset is where you can deploy Integrations developed with WSO2 Micro Integrator as an API. In this preset, you have three different ways to define endpoints. Choreo gives priory to the definition of endpoints in the below-mentioned order. 
```

Comment:
Looks like the i in integrations is mistakenly capitalized.

Reply:
fixed